libr/anal/p/anal_arm_cs.c

```diff
@@ -1801,6 +1801,12 @@ jmp $$ + 4 + ( [delta] * 2 )
 			op->fail = addr + op->size;
 		}
 		break;
+	case ARM_INS_ADR:
+		op->type = R_ANAL_OP_TYPE_LEA;
+		// Set the pointer address and align it
+		op->ptr = IMM(1) + addr + 4 - (addr%4);
```

Comment:
Sorry I think that should have been this link for the PC-relative ADR instruction and Table 10-3 PC-relative offsets http://www.keil.com/support/man/docs/armasm/armasm_dom1361358903373.htm